openpmd_viewer/openpmd_timeseries/data_reader/h5py_reader/field_reader.py

```diff
@@ -288,15 +301,15 @@ def find_dataset( dfile, field_path ):
     return( group, dset )
 
 
-def get_grid_parameters( dfile, avail_fields, metadata ):
+def get_grid_parameters( filename, avail_fields, metadata ):
```

Comment:
`dfile` was an `h5py` filehandle. In order to have `h5py`-agnostic code outside of this function, we now pass a string.